### ISSUE TITLE
Archive failed, error: exit status 65

### ISSUE THREAD

**Issue by oezguercelebi** [bitrise-steplib/steps-xcode-archive#212]
I wanted to set up two workflows. A release and dev build for our QA.

But I am trying now for half a day to fix the issue and I don't understand what the problem is. The log shows my correct name, but later in the logs it looks like that `No certificate for team 'S6XWPU628M' matching 'Apple Development: ázgþr Celebi (359KG8ZNWB)'` . My name is Özgür. I created new provisioning profiles and new certificates and still, it doesn't work.

[61cfd9bc13a6fc34.log](https://github.com/bitrise-steplib/steps-xcode-archive/files/5797550/61cfd9bc13a6fc34.log)

**Comment by bitce** [comment]
Hi @oezguercelebi! Sorry to hear about the issue. Can you send us a link to the app and allow us access? That provides us access to the list of certificates and such that would help us debug this.

**Comment by oezguercelebi** [comment]
Hi @bitce, customer support helped me to resolve the issue. I followed a webinar where a "certificate-and-profile-installer" and "ios-auto-provision" was in the workflow. But that seems to be the issue. I removed now the "ios-auto-provision" and manually added the provision profiles from the codesigndoc export. Now I am waiting for customer support to get back to me, because I am struggling to create .ipa version that uses the debug build configuration. But that has nothing to do with the listed issue here. Access is already granted: https://app.bitrise.io/app/69ac985300b7b6f0#/builds.
